assets/js/modules/analytics/datastore/__fixtures__/create-account.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "abc123",
+  "kind": "analytics#accountTicket",
+  "account": {
+    "name": "Test Account"
+  },
+  "webproperty": {
+    "name": "Test Property"
+  },
+  "profile": {
+    "name": "Test Profile",
+    "timezone": "America/Los Angeles"
+  },
+  "redirectUri": "https://redirect.com/wp-admin?gatoscallback"
```

Comment:
Should this not be:
```suggestion
  "redirectUri": "https://redirect.com/wp-admin?gatoscallback=1"
```

Reply:
Eagle eye! Fixed in https://github.com/google/site-kit-wp/pull/1390/commits/a08b6f47508812cda73b72d292ee95faedb76831.